{{cookiecutter.github_repository}}/settings/common.py

```diff
@@ -125,6 +130,7 @@
 # this middleware classes will be applied in the order given, and in the
 # response phase the middleware will be applied in reverse order.
 MIDDLEWARE = [
+    'corsheaders.middleware.CorsMiddleware',
```

Comment:
Maybe we should have this as an option. This is adding a middleware and potentially decreasing performance for every project that doesn't need CORS @theskumar @jainmickey